packages/stable/src/api/geospatial/types.ts

```diff
@@ -269,3 +269,38 @@ export interface GeospatialCRSResponse
 export interface GeospatialSridId {
   srid: number;
 }
+
+export type GeospatialComputeFunction =
+  | {
+      stTransform: GeospatialGeometryTransformComputeFunction;
+    };
+
+export interface GeospatialGeometryTransformComputeFunction {
+  geospatialGeometryComputeFunction: GeospatialGeometryComputeFunction;
+  srid: number;
+}
+
+export type GeospatialGeometryComputeFunction =
```

Comment:
suggestion(blocking): `GeospatialGeometryComputeFunction` should be just extending `GeospatialGeometryValueComputeFunction`.
Something like

```suggestion
      export interface GeospatialGeometryComputeFunction extends GeospatialGeometryValueComputeFunction {
      }
```

Reply:
I think this has to be the other way around right? Because the `GeospatialGeometryValueComputeFunction` is just one function in the `GeospatialGeometryComputeFunction` group? Or at least it will be when the rest of the compute endpoint is stabilized